packages/web-app-files/src/views/spaces/GenericSpace.vue

```diff
@@ -393,13 +397,36 @@ export default defineComponent({
           performLoaderTask(true, path, fileId)
         }
       )
+      const filesViewWrapper = document.getElementsByClassName('files-view-wrapper')[0]
+      filesViewWrapper?.addEventListener('contextmenu', (event) => {
+        const { target } = event
+        if (
+          (target as HTMLElement).closest('tbody') ||
```

Comment:
I would prefer if you would a) define a new dummy class, b) apply that to both the tbody and all the tile list items and c) use that new dummy class as selector here. That way you don't get a dependency on specific file view modes. If we ever implement a third file view mode it would be sufficient to give the correct item your dummy class to make the whitespace context menu work.